README.md

```diff
@@ -27,6 +27,7 @@ The documentation below describes the newer Go version only.
 
 It uses a simple algorithm:
 - If the environment variable `USE_BAZEL_VERSION` is set, it will use the version specified in the value.
+- Otherwise, if a `.bazeliskrc` file exists in the workspace root and contains the `USE_BAZEL_VERSION` variable, this version will be used.
```

Comment:
@fweikert Should it be noted of the other uses of .bazeliskrc aside from USE_BAZEL_VERSION? Perhaps a general statement of the file, its format, and that all env vars can reside there too?

Reply:
Excellent point - I'll add something